math/Makefile.in

```diff
@@ -55,7 +57,11 @@ umach.o : umach.f
 
 local@LIBPRE@MdsMath = @LIBPRE@MdsMath@SHARETYPE@
 
-FFLAGS += $(FCFLAGS)
+# WARNING: gfortran 10 dangerous fixes ... actual fortran code must be reviewed asap
```

Comment:
so this part is basically unchanged. can we remove the comment to eliminate the diff

Reply:
just a reminder because I doubt in m4 file we will remember it :-)

Reply:
You are right I'll remove it